packages/material-ui/src/Unstable_TrapFocus/TrapFocus.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export interface TrapFocusProps {
+  open: boolean;
```

Comment:
Could you try to move descriptions of the props to the TypeScript definitions?

Reply:
Yes, working on tests and will try to move. Just move descriptions to typescript?